src/testengine.user.storagestate/testengine.user.storagestate.csproj

```diff
@@ -32,7 +36,15 @@
   </ItemGroup>
 
   <ItemGroup>
-    <MySourceFiles Include="..\..\bin\$(configuration)\testengine.user.storagestate\testengine.user.storagestate.dll" />
+    <!-- Add required dependancy assemblies -->
+    <MySourceFiles Include="..\..\bin\$(configuration)\testengine.user.storagestate\testengine.user.storagestate.dll;..\..\bin\$(configuration)\testengine.user.storagestate\Microsoft.AspNetCore.*.dll;..\..\bin\$(configuration)\testengine.user.storagestate\Microsoft.Extensions.DependencyInjection.*.dll;..\..\bin\$(configuration)\testengine.user.storagestate\Microsoft.PowerPlatform.Dataverse.Client.dll;..\..\bin\$(configuration)\testengine.user.storagestate\Microsoft.Xrm.Sdk.dll;..\..\bin\$(configuration)\testengine.user.storagestate\Microsoft.Extensions.Http.dll;..\..\bin\$(configuration)\testengine.user.storagestate\System.ServiceModel.Primitives.dll;..\..\bin\$(configuration)\testengine.user.storagestate\System.Configuration.ConfigurationManager.dll;..\..\bin\$(configuration)\testengine.user.storagestate\Microsoft.Extensions.Caching.*.dll;..\..\bin\$(configuration)\testengine.user.storagestate\Microsoft.Crm.Sdk.Proxy.dll;..\..\bin\$(configuration)\testengine.user.storagestate\Microsoft.Crm.Sdk.Proxy.dll;..\..\bin\$(configuration)\testengine.user.storagestate\Microsoft.Identity*.dll;..\..\bin\$(configuration)\testengine.user.storagestate\System.ServiceModel.*.dll;..\..\bin\$(configuration)\testengine.user.storagestate\System.Security.*.dll;..\..\bin\$(configuration)\testengine.user.storagestate\Microsoft.Bcl.Cryptography.dll" />
```

Comment:
System.Text.RegularExpressions dll copy

Reply:
I could not see System.Text.RegularExpressions.dll in the bin\Release\testengine.user.storagestage folder to copy

Reply:
I meant the package, may have other dlls